src/admin/class-sign-in-with-google-admin.php

```diff
@@ -503,7 +503,10 @@ public function google_auth_redirect() {
 	 */
 	public function authenticate_user() {
 
-		$this->set_access_token( $_GET['code'] );
+		$result = $this->set_access_token( $_GET['code'] );
+		if ( is_wp_error($result) ) {
+			error_log("class-sign-in-with-google-admin.php: WP Error returned from set_access_token. '" . $result->get_error_message(). "'");
```

Comment:
It isn't a good practice to hard code filenames in an error log. It might be better to use something like `__METHOD__`. 

It has been a long time since I worked on this, but why continue if the `set_access_token()` method returns a `WP_Error`? Would it be best to exit here?